### ISSUE TITLE
Add eslint

### ISSUE THREAD

**Issue by LFP6** [EteRNAgame/Curated-boosters#1]
I'd like to start standardizing on using the airbnb eslint config. We could set this up with a CI check to make sure and PRs pass, I think.

**Comment by Omei** [comment]
Is this something you can investigate without (initially) putting any
constraints that might slow down initial development?

**Comment by LFP6** [comment]
Yeah, sure. It's generally something that doesn't require a lot of extra works, but the stylistic consistency is a great benefit.
